### PR TITLE
#32 updating logic for reads qc

### PR DESCRIPTION
**Pull request by vlilanl**
I updated the input.json so that if there are multiple already interleaved files, they go into `rqcfilter.input_files`. If the files are not interleaved, Read 1 will be placed in `rqcfilter.input_fq1` and Read 2 will be placed in `rqcfilter.input_fq2`.  I have also added a boolean to say whether or not the files are interleaved. 

This was tested on JAWS.
Interleaved test of 1 : 83296
Interleaved test of 2 : 83295
Not interleaved test of 2 files(1 interleaved file) : 83439
Not interleaved test of 4 files(2 interleaved files) : 83447

### CODE REVIEW

**Comment by aclum** [comment]
@vlilanl  at least 83296 doesnt appear to correctly run the finish_rqc to generate the final files.

**Comment by vlilanl** [comment]
New tests on jaws:
Interleaved test of 1 : 83868
Interleaved test of 2 : 83870
Not interleaved test of 2 files(1 interleaved file) : 83942
Not interleaved test of 4 files(2 interleaved files) : 83943

@kaijli mentioned that there might be potential memory risks later on especially if the input files are large. She recommends that we update the memory to how she did it in [metat](https://github.com/microbiomedata/metaT_ReadsQC/commit/f866a634a074823137c936e4d870bbff554f00f4#diff-242cc0d86ef4c17f9774dc58b4cb9b81458625082df2808c76b7933df43c57ceR156).

**Review by aclum — APPROVED** [review]

**Comment by vlilanl** [comment]
Jaws test for increased memory: 84325

**Review by kaijli — APPROVED** [review]

File: shortReadsqc.wdl [hunk 1]
```diff
@@ -218,16 +248,16 @@ task rqcfilter {
             File stat = filename_stat
             File stat2 = filename_stat2
             File info_file = filename_reproduce
-            File filtered = glob("filtered/*fastq.gz")[0]
+            File filtered = glob("filtered/*.anqdpht.fastq.gz")[0]
```
Comment (kaijli) [inline shortReadsqc.wdl:251]: remove glob for `File filtered = "filtered/raw.anqdpht.fastq.gz"`
per: https://github.com/microbiomedata/metaT_ReadsQC/pull/8#discussion_r1720178665

Comment (vlilanl) [reply shortReadsqc.wdl]: Jaws test: 84451

Comment (aclum) [reply shortReadsqc.wdl]: looks good

**Review by aclum — APPROVED** [review]

**Pull request merged**